docs/manage/configuration.md

```diff
@@ -149,17 +149,17 @@ Depending on your robot, you may not need to configure any modules, remotes, pro
 
 Components represent the pieces of hardware on your robot that you want to control with Viam.
 
-You must configure each component with a name, a model, a type, attributes, and dependencies:
-
-- `name`: Serves as an identifier when accessing the resource from your code, as well as when configuring other resources that are dependent on that resource.
-You can choose any unique name for a component.
+You must configure each component with a model, a type, a name, attributes, and dependencies:
 
 - `type`: The broad component category, such as `motor`, `arm` or `camera`.
   Components of a given type have a common API.
 
 - `model`: Indicates the more specific category of hardware.
 Components of the same model are supported using the same low-level code.
 
+- `name`: Serves as an identifier when accessing the resource from your code, as well as when configuring other resources that are dependent on that resource.
+You can choose any unique name for a component.
+

```

Comment:
nit: if possible, let's match order of terms in intro sentence to order of terms listed below (i.e. `model`, `type`, `name` order of list items as well)

Reply:
Ooh yes good catch! I almost but didn't do that above; doing now :D